### PR TITLE
Fix/ppwp 1083/8 bin digito

### PR DESCRIPTION
**Pull request by hugodamm**
## Description

Additional information about credit card brands applied to international changes in transactions and previous versions of the Mercado Pago Plugin will be discontinued.

### CODE REVIEW

File: guides/prestashop/introduction.pt.md [hunk 1]
```diff
@@ -9,5 +9,13 @@ Com o plugin do Mercado Pago você pode configurar o pagamento utilizando o [Che
 Também há possibilidade de utilizar o [Checkout Transparente](https://www.mercadopago.[FAKER][URL][DOMAIN]/developers/pt/guides/checkout-api/introduction), em que o pagamento é realizado com cartão de crédito sem sair da sua loja virtual e, o **Ticket Checkout**, com os pagamentos sendo realizados presencialmente com dinheiro ou boleto bancário.
  
 Uma outra característica da integração é a possibilidade de ativação do modo binário, um recurso que permite configurar a aprovação ou a recusa automática de pagamentos. Além disso, os usuários também poderão pagar na sua loja sem estarem obrigatoriamente cadastrados no Mercado Pago.
+
+> WARNING 
+> 
+> Mantenha seu plugin atualizado para não perder vendas
+> 
+> Em outubro de 2022, versões do Plugin Mercado Pago anteriores a 4.3.0 serão descontinuadas e vão parar de funcionar. Além disso, em abril de 2022, as bandeiras de cartões de crédito aplicaram mudanças internacionais nas transações.
```
Comment (hgaldino) [inline guides/prestashop/introduction.pt.md:17]:
```suggestion
> Em outubro de 2022, versões do Plugin Mercado Pago anteriores a 4.3.0 serão descontinuadas e vão parar de funcionar. Além disso, em abril as bandeiras de cartões de crédito aplicaram mudanças internacionais em suas transações.  
```

File: guides/magento-two/previous-requirements.es.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 Para utilizar la integración de Mercado Pago con Magento 2 en tu tienda, debes cumplir con los requisitos que te detallamos a continuación.
 
+> WARNING 
+> 
+> Manten tu plugin actualizado para no perder ventas
```
Comment (romisoto) [inline guides/magento-two/previous-requirements.es.md:7]:
```suggestion
> Mantén tu plugin actualizado para no perder ventas
```

File: guides/magento-two/previous-requirements.es.md [hunk 3]
```diff
@@ -2,6 +2,14 @@
...
+> 
+> En octubre del 2022, las versiones del Plugin Mercado Pago anteriores a las 3.5.0 dejarán de funcionar. Además, en abril del 2022, las tarjetas de crédito aplicarán cambios internacionacionales en las transacciones.
```
Comment (romisoto) [inline guides/magento-two/previous-requirements.es.md:9]:
```suggestion
> En octubre de 2022, las versiones del Plugin Mercado Pago anteriores a la 3.5.0 dejarán de funcionar. Además, en abril de 2022, las tarjetas de crédito aplicarán cambios internacionales en las transacciones.  
```

File: guides/magento-two/previous-requirements.es.md [hunk 4]
```diff
@@ -2,6 +2,14 @@
 
 Para utilizar la integración de Mercado Pago con Magento 2 en tu tienda, debes cumplir con los requisitos que te detallamos a continuación.
 
+> WARNING 
+> 
+> Manten tu plugin actualizado para no perder ventas
+> 
+> En octubre del 2022, las versiones del Plugin Mercado Pago anteriores a las 3.5.0 dejarán de funcionar. Además, en abril del 2022, las tarjetas de crédito aplicarán cambios internacionacionales en las transacciones.  
+>
+> **Manten tu tienda siempre actualizada con la versión más reciente.**
```
Comment (romisoto) [inline guides/magento-two/previous-requirements.es.md:11]:
```suggestion
> **Mantén tu tienda siempre actualizada con la versión más reciente.**
```

File: guides/prestashop/introduction.es.md [hunk 5]
```diff
@@ -9,6 +9,14 @@ Con el plugin Mercado Pago puedes configurar el pago usando [Checkout Pro](https
 También existe la posibilidad de utilizar el [Checkout API](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/checkout-api/introduction), con el cual el pago se realiza con tarjeta de crédito sin salir de tu tienda virtual y del **Ticket Checkout**, realizándose los pagos en persona con efectivo o comprobante bancario.
  
 Otra característica de la integración es la posibilidad de activar el modo binario, característica que permite configurar la aprobación o rechazo automático de pagos. Además, los usuarios también podrán pagar en tu tienda sin estar obligatoriamente registrados en Mercado Pago.
+
+> WARNING 
+> 
+> Manten tu plugin actualizado para no perder ventas
```
Comment (romisoto) [inline guides/prestashop/introduction.es.md:15]:
```suggestion
> Mantén tu plugin actualizado para no perder ventas
```

File: guides/prestashop/introduction.es.md [hunk 6]
```diff
@@ -9,6 +9,14 @@ Con el plugin Mercado Pago puedes configurar el pago usando [Checkout Pro](https
...
+> 
+> En octubre del 2022, las versiones del Plugin Mercado Pago anteriores a las 4.3.0 dejarán de funcionar. Además, en abril del 2022, las tarjetas de crédito aplicarán cambios internacionacionales en las transacciones.
```
Comment (romisoto) [inline guides/prestashop/introduction.es.md:17]:
```suggestion
> En octubre de 2022, las versiones del Plugin Mercado Pago anteriores a la 4.3.0 dejarán de funcionar. Además, en abril de 2022, las tarjetas de crédito aplicarán cambios internacionales en las transacciones. 
```

File: guides/prestashop/introduction.es.md [hunk 7]
```diff
@@ -9,6 +9,14 @@ Con el plugin Mercado Pago puedes configurar el pago usando [Checkout Pro](https
...
  
 Otra característica de la integración es la posibilidad de activar el modo binario, característica que permite configurar la aprobación o rechazo automático de pagos. Además, los usuarios también podrán pagar en tu tienda sin estar obligatoriamente registrados en Mercado Pago.
+
+> WARNING 
+> 
+> Manten tu plugin actualizado para no perder ventas
+> 
+> En octubre del 2022, las versiones del Plugin Mercado Pago anteriores a las 4.3.0 dejarán de funcionar. Además, en abril del 2022, las tarjetas de crédito aplicarán cambios internacionacionales en las transacciones. 
+>
+> **Manten tu tienda siempre actualizada con la versión más reciente.**
```
Comment (romisoto) [inline guides/prestashop/introduction.es.md:19]:
```suggestion
> **Mantén tu tienda siempre actualizada con la versión más reciente.**
```

File: guides/woocommerce/introduction.es.md [hunk 8]
```diff
@@ -2,7 +2,15 @@
...
 
-Para obtener más información sobre los beneficios, tarifas, términos y tipos de pago disponibles, visita la página [WooCommerce con Mercado Pago](https://www.mercadopago.com.br/ferramentas-para-vender/link-de-pagamento-plugins-checkout/plugins/vender-woocommerce?utm_experiment=optimize&matt_tool=10047855&matt_word=MLB_MP_G_AO_OP_COW_SEARCH_SELL_TXS_Checkout-WooCommerce-Branded&gclid=Cj0KCQjw6s2IBhCnARIsAP8RfAjJHJVBE0is6oJv5K3_v9qiv44mYs52j7TlFtAMe2AVWKoJ-DAga_4aAuH7EALw_wcB). 
+Para obtener más información sobre los beneficios, tarifas, términos y tipos de pago disponibles, visita la página [WooCommerce con Mercado Pago](https://www.mercadopago.com.br/ferramentas-para-vender/link-de-pagamento-plugins-checkout/plugins/vender-woocommerce?utm_experiment=optimize&matt_tool=10047855&matt_word=MLB_MP_G_AO_OP_COW_SEARCH_SELL_TXS_Checkout-WooCommerce-Branded&gclid=Cj0KCQjw6s2IBhCnARIsAP8RfAjJHJVBE0is6oJv5K3_v9qiv44mYs52j7TlFtAMe2AVWKoJ-DAga_4aAuH7EALw_wcB).
+
+> WARNING 
+> 
+> Manten tu plugin actualizado para no perder ventas
```
Comment (romisoto) [inline guides/woocommerce/introduction.es.md:9]:
```suggestion
> Mantén tu plugin actualizado para no perder ventas
```

File: guides/woocommerce/introduction.es.md [hunk 9]
```diff
@@ -2,7 +2,15 @@
...
+> 
+> En octubre del 2022, las versiones del Plugin Mercado Pago anteriores a las 4.5.0 dejarán de funcionar. Además, en abril del 2022, las tarjetas de crédito aplicarán cambios internacionacionales en las transacciones.
```
Comment (romisoto) [inline guides/woocommerce/introduction.es.md:11]:
```suggestion
> En octubre de 2022, las versiones del Plugin Mercado Pago anteriores a la 4.5.0 dejarán de funcionar. Además, en abril de 2022, las tarjetas de crédito aplicarán cambios internacionales en las transacciones.  
```

File: guides/woocommerce/introduction.es.md [hunk 10]
```diff
@@ -2,7 +2,15 @@
...
+Para obtener más información sobre los beneficios, tarifas, términos y tipos de pago disponibles, visita la página [WooCommerce con Mercado Pago](https://www.mercadopago.com.br/ferramentas-para-vender/link-de-pagamento-plugins-checkout/plugins/vender-woocommerce?utm_experiment=optimize&matt_tool=10047855&matt_word=MLB_MP_G_AO_OP_COW_SEARCH_SELL_TXS_Checkout-WooCommerce-Branded&gclid=Cj0KCQjw6s2IBhCnARIsAP8RfAjJHJVBE0is6oJv5K3_v9qiv44mYs52j7TlFtAMe2AVWKoJ-DAga_4aAuH7EALw_wcB).
+
+> WARNING 
+> 
+> Manten tu plugin actualizado para no perder ventas
+> 
+> En octubre del 2022, las versiones del Plugin Mercado Pago anteriores a las 4.5.0 dejarán de funcionar. Además, en abril del 2022, las tarjetas de crédito aplicarán cambios internacionacionales en las transacciones.  
+>
+> **Manten tu tienda siempre actualizada con la versión más reciente.**
```
Comment (romisoto) [inline guides/woocommerce/introduction.es.md:13]:
```suggestion
> **Mantén tu tienda siempre actualizada con la versión más reciente.**
```

**Review by romisoto — CHANGES_REQUESTED** [review]
solo unos pequeños ajustes

File: guides/woocommerce/introduction.pt.md [hunk 11]
```diff
@@ -4,6 +4,14 @@
 
 Para mais informações sobre os benefícios, tarifas, prazos e tipos de checkout disponíveis, acesse a página [WooCommerce com Mercado Pago](https://www.mercadopago.com.br/ferramentas-para-vender/link-de-pagamento-plugins-checkout/plugins/vender-woocommerce?utm_experiment=optimize&matt_tool=10047855&matt_word=MLB_MP_G_AO_OP_COW_SEARCH_SELL_TXS_Checkout-WooCommerce-Branded&gclid=Cj0KCQjw6s2IBhCnARIsAP8RfAjJHJVBE0is6oJv5K3_v9qiv44mYs52j7TlFtAMe2AVWKoJ-DAga_4aAuH7EALw_wcB). 
 
+> WARNING 
+> 
+> Mantenha seu plugin atualizado para não perder vendas
+> 
+> Em outubro de 2022, versões do Plugin Mercado Pago anteriores a 4.5.0 serão descontinuadas e vão parar de funcionar. Além disso, em abril de 2022, as bandeiras de cartões de crédito aplicaram mudanças internacionais nas transações.
```
Comment (hgaldino) [inline guides/woocommerce/introduction.pt.md:11]:
```suggestion
> Em outubro de 2022, versões do Plugin Mercado Pago anteriores a 4.5.0 serão descontinuadas e deixarão de funcionar. Além disso, em abril do mesmo ano as bandeiras de cartões de crédito aplicaram mudanças internacionais nas transações.  
```

**Review by romisoto — APPROVED** [review]

File: guides/magento-two/previous-requirements.pt.md [hunk 12]
```diff
@@ -2,6 +2,14 @@
 
 Para usar a integração do Mercado Pago com o Magento 2 em sua loja, você deve atender aos requisitos descritos na tabela abaixo.
 
+> WARNING 
+> 
+> Mantenha seu plugin atualizado para não perder vendas
+> 
+> Em outubro de 2022, versões do Plugin Mercado Pago anteriores a 3.5.0 serão descontinuadas e vão parar de funcionar. Além disso, em abril de 2022, as bandeiras de cartões de crédito aplicaram mudanças internacionais nas transações.
```
Comment (hgaldino) [inline guides/magento-two/previous-requirements.pt.md:9]:
```suggestion
> Em outubro de 2022, as versões do plugin Mercado Pago anteriores a 3.5.0 serão descontinuadas e deixarão de funcionar. Além disso, em abril as bandeiras de cartões de crédito aplicaram mudanças internacionais nas transações.  
```

**Review by hgaldino — APPROVED** [review]

**Pull request merged**